src/routes/pratio/+page.svelte

```diff
@@ -0,0 +1,84 @@
+<script lang="ts">
+  import { RangeSlider } from '@skeletonlabs/skeleton';
+	import { determinePRatio } from '../../utils/PRatioDeterminer';
+
+  let sex ='male';
+  let fitnessLevel = 'beginner'
+  let calorieSurplus = 10;
+
+  let calorieSurplusRange = [2.5, 5, 10, 15, 25]
+
+  let bodyFat = 10;
+  let bodyFatMin = 4;
+  let bodyFatMax = 20;
+
+  $: if(sex === 'male'){
+    bodyFat = 10;
+    bodyFatMin = 4;
+    bodyFatMax = 20
+  }
+  else{
+    bodyFat = 22
+    bodyFatMin = 10;
+    bodyFatMax = 32;
```

Comment:
bodyfat max for female is 40%

https://github.com/thinklikeadesigner/FlexFactor/issues/19
```suggestion
    bodyFatMax = 40;
```

Reply:
Good catch !!